extensions/couchbase/processors/GetCouchbaseKey.h

```diff
@@ -0,0 +1,113 @@
+    "the entire FlowFile will be buffered in memory.";
+
+  EXTENSIONAPI static constexpr auto CouchbaseClusterControllerService = core::PropertyDefinitionBuilder<>::createProperty("Couchbase Cluster Controller Service")
+      .withDescription("A Couchbase Cluster Controller Service which manages connections to a Couchbase cluster.")
+      .withAllowedTypes<controllers::CouchbaseClusterService>()
+      .isRequired(true)
+      .build();
+  EXTENSIONAPI static constexpr auto BucketName = core::PropertyDefinitionBuilder<>::createProperty("Bucket Name")
+      .withDescription("The name of bucket to access.")
+      .withDefaultValue("default")
+      .isRequired(true)
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto ScopeName = core::PropertyDefinitionBuilder<>::createProperty("Scope Name")
+      .withDescription("Scope to use inside the bucket. If not specified, the _default scope is used.")
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto CollectionName = core::PropertyDefinitionBuilder<>::createProperty("Collection Name")
+      .withDescription("Collection to use inside the bucket scope. If not specified, the _default collection is used.")
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto DocumentType = core::PropertyDefinitionBuilder<3>::createProperty("Document Type")
+      .withDescription("Content type of the retrieved value.")
+      .isRequired(true)
+      .withDefaultValue(magic_enum::enum_name(CouchbaseValueType::Json))
+      .withAllowedValues(magic_enum::enum_names<CouchbaseValueType>())
+      .build();
+  EXTENSIONAPI static constexpr auto DocumentId = core::PropertyDefinitionBuilder<>::createProperty("Document Id")
+      .withDescription("A static, fixed Couchbase document id, or an expression to construct the Couchbase document id.")
+      .supportsExpressionLanguage(true)
+      .build();
+  EXTENSIONAPI static constexpr auto PutValueToAttribute = core::PropertyDefinitionBuilder<>::createProperty("Put Value to Attribute")
+      .withDescription("If set, the retrieved value will be put into an attribute of the FlowFile instead of a the content of the FlowFile. "
+                       "The attribute key to put to is determined by evaluating value of this property.")
+      .supportsExpressionLanguage(true)
+      .build();
+
+  EXTENSIONAPI static constexpr auto Properties = std::to_array<core::PropertyReference>({
+    CouchbaseClusterControllerService,
+    BucketName,
+    ScopeName,
+    CollectionName,
+    DocumentType,
+    DocumentId,
+    PutValueToAttribute
+  });
+
+  EXTENSIONAPI static constexpr auto Success = core::RelationshipDefinition{"success",
+    "Values retrieved from Couchbase Server are written as outgoing FlowFiles content or put into an attribute of the incoming FlowFile and routed to this relationship."};
```

Comment:
```suggestion
  EXTENSIONAPI static constexpr auto Success = core::RelationshipDefinition{"success",
      "Values retrieved from Couchbase Server are written as outgoing FlowFiles content or put into an attribute of the incoming FlowFile and routed to this relationship."};
```

Reply:
Updated in https://github.com/apache/nifi-minifi-cpp/pull/1881/commits/de69b7fd973b0333b18c97b5764396ca0fe77fe0